tsconfig.json

```diff
@@ -8,8 +8,7 @@
     // Output incremental build info to node_modules
     "outDir": "node_modules/.cache/typescript",
     "incremental": true,
-    "resolveJsonModule": true,
-    "types": ["vitest-puppeteer"]
```

Comment:
This didn't seem to actually do anything. In the one file that `tsc` complained about, `test-utils/storybook.ts`, a side-effect import of `import 'vitest-puppeteer'` was enough to inject the necessary global types.